pkg/api/serialization_test.go

```diff
@@ -122,8 +121,8 @@ func roundTripSame(t *testing.T, item runtime.Object, except ...string) {
 
 // For debugging problems
 func TestSpecificKind(t *testing.T) {
-	api.Scheme.Log(t)
-	defer api.Scheme.Log(nil)
+	// api.Scheme.Log(t)
```

Comment:
Add a comment explaining why you leave this code commented instead of removing it. Same below.